fieldImages/src/main/java/edu/wpi/first/fields/IField.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.fields;
+
+public interface IField {
```

Comment:
WPILib doesn't use `I` to indicate interfaces and `Field` is available, so this should probably be renamed to `Field`.